google/cloud/spanner/samples/samples.cc

```diff
@@ -3248,11 +3248,13 @@ std::string PickConfig(google::cloud::spanner::InstanceAdminClient client,
   std::cout << "\n" << __func__ << ":\n";
 
   std::string ret;
+  std::string first;
   int i = 0;
   for (auto const& instance_config : client.ListInstanceConfigs(project_id)) {
     if (!instance_config) break;
     if (instance_config->name().rfind(config_prefix, 0) != 0) continue;
     auto const config = instance_config->name().substr(config_prefix.size());
+    if (first.empty()) first = config;
```

Comment:
I am not sure you need another `first` variable if you change this to `if (ret.empty()) ret = config;` ?

Reply:
Yes, but I thought that was less clear.